stytch/api/users.py

```diff
@@ -60,8 +60,8 @@ def delete(self, user_id: str):
     def update(
         self,
         user_id: str,
-        emails: Optional[list[str]] = None,
-        phone_numbers: Optional[list[str]] = None,
+        emails: 'Optional[list[str]]' = None,
+        phone_numbers: 'Optional[list[str]]' = None,
```

Comment:
Nice find! `mypy` doesn't the stringification, but this should work instead: 
```suggestion
        emails: Optional[List[str]] = None,
        phone_numbers: Optional[List[str]] = None,
```
with an extra import up top:
```diff
-from typing import Dict, Optional
+from typing import Dict, List, Optional
```

Reply:
Ah, yup, that makes sense! Thanks!